src/babashka/curl.clj

```diff
@@ -174,23 +175,39 @@
                   :headers headers
                   :body body
                   :err err
-                  :process (:proc opts)}]
+                  :process process
+                  :exit exit}]
     response))
 
 ;;;; End Response Parsing
 
+(def unexceptional-status?
+  #{200 201 202 203 204 205 206 207 300 301 302 303 304 307})
```

Comment:
Pulled this from https://github.com/dakrone/clj-http/blob/3.x/src/clj_http/client.clj#L197-L199